File: atividade_5/raiz.c
#include <stdio.h>
#include <math.h>

double calcular_raiz_quadrada(double n) {
    double b = 2.0; 
    while (fabs(n - b * b) > 0.0001) {
        double p = (b + (n / b)) / 2.0;
        b = p;
    }
    return b;
}

int main() {
    double numero, raiz_quadrada;

    printf("Digite o numero para calcular a raiz quadrada: ");
    scanf("%lf", &numero);

    raiz_quadrada = calcular_raiz_quadrada(numero);

    printf("A raiz quadrada de %.4lf é aproximadamente %.4lf\n", numero, raiz_quadrada);

    return 0;
}

File: atividade_5/raiz.py
def calcular_raiz_quadrada(n):
    b = 2 
    while abs(n - b * b) > 0.0001:
        p = (b + (n / b)) / 2
        b = p
    return b

numero = float(input("Digite o número para calcular a raiz quadrada: "))
raiz_quadrada = calcular_raiz_quadrada(numero)
print(f"A raiz quadrada de {numero} é aproximadamente {raiz_quadrada:.4f}")
